input/input.c

```diff
@@ -1693,8 +1720,11 @@ void mp_input_src_feed_cmd_text(struct mp_input_src *src, char *buf, size_t len)
                 bstr s = {in->cmd_buffer, in->cmd_buffer_size};
                 s = bstr_strip(s);
                 struct mp_cmd *cmd = mp_input_parse_cmd_str(src->log, s, "<>");
-                if (cmd)
-                    mp_input_queue_cmd(src->input_ctx, cmd);
+                if (cmd) {
+                    input_lock(src->input_ctx);
+                    queue_cmd(src->input_ctx, cmd);
+                    input_unlock(src->input_ctx);
+                }
```

Comment:
I understand why we prefer calling internal static variant, but maybe to avoid duplication we should just call `mp_input_queue_cmd`? Or add static `queue_cmd_locked` and `queue_cmd_unlocked`? (one of them could be without suffix)

``` suggestion
                mp_input_queue_cmd(ictx, cmd);
```

minor nitpick, I don't mind either way, if you prefer current approach.